examples/dreamcast/vmu/vmu_beep/beep.c

```diff
@@ -39,18 +40,25 @@
 
 KOS_INIT_FLAGS(INIT_DEFAULT);
```

Comment:
```suggestion
```
Might as well get rid of it, since it's no longer needed for the defaults.

Reply:
It was never needed for the defaults. Don't remember why it was tossed in.